ddtrace/contrib/logging/patch.py

```diff
@@ -1,19 +1,25 @@
 import logging
 from wrapt import wrap_function_wrapper as _w
 
+from ddtrace import config
+
 from ...helpers import get_correlation_ids
 from ...utils.wrappers import unwrap as _u
 
 RECORD_ATTR_TRACE_ID = 'dd.trace_id'
 RECORD_ATTR_SPAN_ID = 'dd.span_id'
 RECORD_ATTR_VALUE_NULL = 0
 
+config._add('logging', dict(
+    # tracer=None by default, override here for custom tracer
```

Comment:
Why is this commented out?

Reply:
Uncommented line -- had commented it after I realized the config api returns `None` by default for attributes that are not found. But I also see the benefit of having specified what the expected configuration options are for an integration.